src/engine/controls/cuecontrol.cpp

```diff
@@ -1746,13 +2114,20 @@ HotcueControl::HotcueControl(QString group, int i)
 
 HotcueControl::~HotcueControl() {
     delete m_hotcuePosition;
+    delete m_hotcueLength;
```

Comment:
I think all of them can become unique_ptr

Reply:
This doesn't work with `connect()`. I could probably use `ptr.get()`, but does this even make sense?

Reply:
It is OK for me to keep the plain pointer here if you like. 
In general we are working on get rid of all plain pointers from new calls. 
ptr.get() for connect() is a solution. We use always plain pointers if the called function promises to not store the pointer. This is probably not the case for connect() but Qt takes care for that so we can ignore it.